portal/models/questionnaire_response.py

```diff
@@ -370,7 +370,8 @@ def quote_double_quote(value):
                         answer['valueCoding'].get('text')
                     )
 
-                    text_and_coded_answers.append({'valueString': text_answer})
+                    if text_answer is not None:
```

Comment:
@ivan-c this change was necessary to get tests to pass.  i can't imagine it's be work ever saving a `valueString` with a `None`, would it?  those were showing up as differences in the following test:
```
 py.test tests/test_assessment_engine.py::TestAssessmentEngine::test_update_assessment
 ```

Reply:
> i can't imagine it would ever be worth saving a `valueString` with a value of `None` in a questionnaire response, would it?

probably not- `None`/null isn't the same type as valueString anyway